README.md

```diff
@@ -467,9 +485,9 @@ The above mentioned method returns an Interaction path and a corresponding Inter
 	```
 
 
-### Send Properties to ONE
+### Send Properties to Thunderhead ONE or Salesforce Interaction Studio
 
-Properties in the form of key/value pair strings can be sent to ONE using the SDK's public methods. Simply create a dictionary of key/value pair strings and call the appropriate properties public method, as follows:
+Properties in the form of key/value pair strings can be sent to Thunderhead ONE or Salesforce Interaction Studio using the SDK's public methods. Simply create a dictionary of key/value pair strings and call the appropriate properties public method, as follows:
```

Comment:
Properties can be sent to Thunderhead ONE or Salesforce Interaction Studio as Strings, in the form of key/value pairs using the SDK's public methods.